### ISSUE TITLE
A11y_AzureDataStudio_Filter_Keyboard: Keyboard focus is not visible on "Ok" button

### ISSUE THREAD

**Issue by SaisreeDandamudi** [microsoft/azuredatastudio#15997]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** 
#A11yMAS; #A11yTCS; #SH-AzureDataStudio-June2021; #SQL Azure Data Studio; #Win10; #DesktopApp; #WCAG2.4.7; #AINotInScope; #Keyboard

### Environment Details:
- Azure Data Studio Version: 1.30.0 (user setup)
- OS Version: 21H1

### Steps to Reproduce:

1. Launch Azure Data Studio
2. Connect to any SQL server and write a query
3. Tab till results grid and press F3 so that the filters pop-up will open
4. Verify whether keyboard focus is clearly visible on "Ok" button

### Actual:
Keyboard focus is not visible on Ok button

### Expected:
The keyboard focus should be clearly visible on "Ok" button

### User Impact:
Keyboard users will not be able to know on which control the focus is if focus is not visible on "Ok" button

**Comment by Charles-Gagnon** [comment]
@alanrenmsft

**Comment by alanrenmsft** [comment]
this is exactly why we should have screenshots when you open the issue, this is what I am seeing: the focus outline is pretty clear

![image](https://user-images.githubusercontent.com/13777222/126021366-212685b1-dd35-4a96-8d6e-0d1fccb96b3d.png)
![image](https://user-images.githubusercontent.com/13777222/126021379-d9d7db3a-7d98-4057-a9a5-a5920e3b725d.png)

**Comment by v-supupp** [comment]
#A11yMAS;#A11yTCS;#SH-AzureDataStudio-June2021;#SQL Azure Data Studio;#Win10;#DesktopApp; #WCAG2.4.7;#AINotInScope; #Keyboard;A11ySev2;

**Comment by v-mtatikonda** [comment]
#A11ySev2;

**Comment by akashelijah05** [comment]
#Closed; #Regressed:06-23-23;

Verified issue in below environment :

Issue is fixed

Version: 1.44.1 (user setup)
Commit: 8f53a316fa00a98264f1ab119641cd540b5af25c
Date: 2023-06-01T02:12:48.765Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
![Ok button focus visible](https://github.com/microsoft/azuredatastudio/assets/116360471/a595b9cb-13d6-47e4-a6e8-643ed6d897ef)
